frontend/turbo.json

```diff
@@ -1,29 +1,16 @@
 {
     "$schema": "https://turborepo.org/schema.json",
     "globalEnv": [
-        "INTERNET_IDENTITY",
-        "NFID_URL",
         "DFX_NETWORK",
-        "IC_URL",
-        "II_DERIVATION_ORIGIN",
-        "CUSTOM_DOMAINS",
-        "STORAGE_INDEX_CANISTER",
-        "LEDGER_CANISTER_ICP",
-        "LEDGER_CANISTER_SNS1",
-        "LEDGER_CANISTER_BTC",
-        "LEDGER_CANISTER_CHAT",
-        "LEDGER_CANISTER_KINIC",
-        "LEDGER_CANISTER_HOTORNOT",
-        "LEDGER_CANISTER_GHOST",
-        "BLOB_URL_PATTERN",
         "NODE_ENV",
         "USERGEEK_APIKEY",
         "METERED_APIKEY",
         "ROLLBAR_ACCESS_TOKEN",
         "TRANSLATE_API_KEY",
         "PUBLIC_TRANSLATE_API_KEY",
         "GIPHY_APIKEY",
-        "FRAME_ANCESTORS"
+        "FRAME_ANCESTORS",
```

Comment:
FRAME_ANCESTORS should not be in here any more.

Reply:
Once your PR is in I'll pull latest into here and remove it during merge conflict resolution